igvc_navigation/src/mapper/map_utils.cpp

```diff
@@ -0,0 +1,231 @@
+#include <unordered_set>
+
+#include <pcl/filters/extract_indices.h>
+#include <pcl/filters/passthrough.h>
+#include <pcl/sample_consensus/method_types.h>
+#include <pcl/sample_consensus/model_types.h>
+#include <pcl/segmentation/sac_segmentation.h>
+
+#include "map_utils.h"
+
+namespace MapUtils
+{
+inline int discretize(radians angle, double angular_resolution)
+{
+  double coeff = 1 / angular_resolution;
+  return static_cast<int>(std::round(coeff * angle));
+}
+
+void filterPointsBehind(const pcl::PointCloud<pcl::PointXYZ>& pc, pcl::PointCloud<pcl::PointXYZ>& filtered_pc,
+                        BehindFilterOptions options)
+{
+  static double start_angle = -M_PI + options.angle / 2;
```

Comment:
what coordinate system is this method using? Is it right-handed or left handed? kind of hard to tell

Reply:
REP103